lib/mixins/link.js

```diff
@@ -92,10 +92,10 @@ export default {
         linkClick(e) {
             if (!this.disabled) {
                 this.$root.$emit('clicked::link', this);
-                this.$emit('click');
+                this.$emit('click', e);
             }
 
-            if (this.disabled || (!this.isRouterLink && this._href === '#')) {
```

Comment:
This was to prevent scroll to top when clicking '#' links ( dropdown items with @click handler for example)